dev/com.ibm.ws.microprofile.rest.client_fat/bnd.bnd

```diff
@@ -26,15 +26,17 @@ src: \
 
 fat.project: true
 
+tested.features=mpRestClient-1.2,jdbc-4.2
```

Comment:
mpRestClient tests jdbc?

Reply:
No, but when using the `RepeatTests` rule, it requires JDBC-4.2 to be specified - I'm guessing it is because of the EE level changes.